parkings/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .addresses import *
+from .operators import *
+from .parkings import *
```

Comment:
Odd, I've seen the complete opposite ;) For example if the `parking.py` would contain in theory many different parking models, would you still use singular?

But changing for this 👍